cmd/scaffold.go

```diff
@@ -171,10 +171,41 @@ func ScaffoldOverrides() {
 
 func executeScaffoldRun(ccmd *cobra.Command, args []string) {
 
+	// initialise a counter to see how many flags have been set
+	flagCount := 0
+
+	// set a list of the flags that we are interested in
+	flags := []string{
```

Comment:
We might need some docs on this so people know when this list needs updating and where the list is?

Reply:
This is a list that is specific to the scaffold operation of the CLI. It is a private method.
However I will update the docs for the scaffold command so that it shows which parameters are required, either on the command line or in a configuration file.